docs/about/migration/23-01.ipynb

```diff
@@ -0,0 +1,193 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "921d565c-0e5a-4a9e-9467-b3d066d08f5c",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "\n",
+    "Starting with Scipp 23.01.0, a `VariancesError` is raised when an operand with variances (statistical uncertianties) is implicitly or explicitly broadcast.\n",
+    "This is a necessary bugfix, since Scipp is not capable of tracking correlations and would thus silently underestimate uncertainties."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6eaaff9a-e08d-4e84-a299-0bbf4ff56b3e",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## What is affected?\n",
+    "\n",
+    "Any operation with a broadcast of an operand with variances is affected.\n",
+    "This can happen for dense data as well as for binned data.\n",
+    "We give two examples:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7ee57dea-da15-439b-8cde-17924a8f3289",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipp as sc\n",
+    "\n",
+    "var = sc.ones(dims=['x', 'y'], shape=(4, 3))\n",
+    "norm = sc.array(\n",
+    "    dims=['y'],\n",
+    "    values=[0.1, 0.2, 0.3],\n",
+    "    variances=[0.1, 0.2, 0.3],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a3501a6-e855-4667-b2fc-a10a5a8bb669",
+   "metadata": {},
+   "source": [
+    "An operation between `var` and `norm` will raise `VariancesError` since `norm` is broadcast into dimension `'x'`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f6f16545-caa4-453d-9e67-18663ca5281e",
+   "metadata": {
+    "tags": [
+     "raises-exception"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "var / norm"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e0bd3114-3502-424f-a70b-ee7306264fae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binned = sc.data.table_xyz(100).bin(y=3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65722b75-6aba-4a30-a9e3-2cbe1eb23394",
+   "metadata": {},
+   "source": [
+    "An operation between `binned` and `norm` will raise `VariancesError` since `norm` is broadcast to all bin elements:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c33fce46-a1be-41aa-9660-c69b83a70548",
+   "metadata": {
+    "tags": [
+     "raises-exception"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "binned / norm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9dd8781c-6003-4243-a1e3-810a4088219d",
+   "metadata": {},
+   "source": [
+    "## Working around the new behavior\n",
+    "\n",
+    "Scipp will not be able to support an automatic approach for handling these cases correctly, since this would be either very costly or very problem-specific.\n",
+    "An easy **but wrong** work around (equivalent to the old behavior in almost all cases) is to set the variances of the term being broadcast to `None`:"
```

Comment:
Not sure this is equivalent to the old behaviour outside of neutron scattering.

Reply:
I think it is equivalent as long as the broadcast is to more than a handful of elements, regardless of neutron-scattering or not?

Reply:
Doesn't it depend on the relative size of variances of the operands?

Reply:
If the other operand has no variances, then a broadcast to 2 elements will suppress errors by a factor of sqrt(2) in the old implementation. This is wrong, and setting it to 0 is more wrong, but I feel this is a rare edge case, i.e., covered ny "almost all cases"? Once you get to 10 or more elements the suppression is so close to setting things to 0 that I would call it "equally wrong"?